tests/test_meshing_workflow.py

```diff
@@ -260,18 +260,17 @@ def test_accessors_for_argument_sub_items(new_mesh_session):
     ).CommandArguments.CadImportOptions.OneZonePer.is_read_only()
     assert (
         w.task(
-            "Import Geometry"
-        ).CommandArguments.CadImportOptions.OneZonePer.default_value()
-        == "Body"
+            "Generate the Volume Mesh"
+        ).CommandArguments.VolumeFillControls.Type.default_value()
+        == "Cartesian"
```

Comment:
I have chosen a different parameter here because the other one has changed in the meshing application itself, and we don't want to handle the differences between the versions here.